_docker/drupal/drupal-filesystem/web/modules/custom/rhd_common/rhd_common.updates.inc

```diff
@@ -0,0 +1,29 @@
+<?php
```

Comment:
@jordanpagewhite Is this the preferred way to do database migrations in Drupal?

If yes then I'm happy to deprecate and ultimately remove the `phinx` DB migration framework that we have (assuming you're on board with that?)

When will this run? Is it applied as part of the `drush updb` command? If yes, I assume the failure of the update will cause the `drush updb` command to fail?

The other question I have is how does Drupal know that this update has been applied? Is there some internal state management somewhere that says `rhd_common_update_8001` has already been done?

Reply:
Thanks for the explanation!